### ISSUE TITLE
Update Email invite text

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/platform-ui#1230]
In the email invitation, please change this sentence
Joining a program means you will have permission to access program data, provided that you also have DACO approval. 

To:
**Joining a program means you will have permission to access, submit or download program data, depending on your program role**

**Comment by blabadi** [comment]
this need to be in program service, not UI

**Comment by kcullion** [comment]
Yes @blabadi @rosibaj I think I need someone to move that. I don't think I can
